### ISSUE TITLE
Migrate to a WebExtension

### ISSUE THREAD

**Issue by mgol** [firebug/firequery#42]
Firefox will not support old-style XUL-based extensions starting November 2017 (in Firefox 57+). Do you have plans to migrate to a new WebExtension standard?

**Comment by janodvarko** [comment]
Yes, but there is more existing extension that need migration too so, this one might take a time (plus I suspect that some new WebExtension API for DevTools will be required for this extension).

Honza
